doc/classes/Shader.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Shader" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A shader implemented in the Godot shading language.
+		A shader implemented in GDShader.
 	</brief_description>
 	<description>
-		A custom shader program implemented in the Godot shading language, saved with the [code].gdshader[/code] extension.
+		A custom shader program implemented in GDShader, saved with the [code].gdshader[/code] extension.
```

Comment:
```suggestion
		A custom shader program implemented in Godot's shading language. GDShader, saved with the [code].gdshader[/code] extension.
```
In a few places we should use this phrasing. Here, and in the various introductions to shaders in the manual.

Also this may be technically incorrect - `Shader` is the base class for visual shaders, too. And while internally there is a GDShader text representation of the visual shader, we may want different phrasing in this description that encompasses both text and visual shaders.